src/adapter/src/optimize/index.rs

```diff
@@ -110,10 +112,6 @@ impl GlobalMirPlan {
     pub fn df_desc(&self) -> &MirDataflowDescription {
         &self.df_desc
     }
-
-    pub fn df_meta(&self) -> &DataflowMetainfo {
```

Comment:
This and all other similar diffs are just removing getters that are no longer used anywhere in the code.